module/views/header_element.tpl

```diff
@@ -70,7 +82,7 @@
          <a id="hosts-states-popover"
             class="hosts-all" data-count="{{ h['nb_elts'] }}" data-problems="{{ h['nb_problems'] }}"
             href="/all?search=type:host"
-            data-original-title="Hosts states" data-toggle="popover popover-hosts" title="Overall hosts states: {{h['nb_elts']}} hosts, {{h["nb_problems"]}} problems" data-html="true" data-trigger="hover">
```

Comment:
@ace13 We need the `data-html="true"` here. Without it, my browser display the html code and don't interpret it.

I keep the rest of the PR which is nice, but I'm adding this back. If you removed it for a good reason, please tell me why.